tests/rptest/tests/e2e_topic_recovery_test.py

```diff
@@ -0,0 +1,159 @@
+# Copyright 2021 Vectorized, Inc.
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.md
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0
+
+from rptest.services.cluster import cluster
+from ducktape.mark import matrix
+from ducktape.cluster.cluster_spec import ClusterSpec
+from rptest.clients.types import TopicSpec
+from rptest.services.redpanda import RedpandaService, SISettings
+from rptest.util import Scale
+from rptest.clients.rpk import RpkTool
+from rptest.tests.redpanda_test import RedpandaTest
+from rptest.services.franz_go_verifiable_services import FranzGoVerifiableProducer, FranzGoVerifiableSeqConsumer
+from rptest.util import wait_until
```

Comment:
nit: `wait_until` is defined in `ducktape.utils.util` - I think it transitively gets imported this way because `rptest.util` imported it.